.github/workflows/build-and-publish.yml

```
@@ -2,10 +2,12 @@ name: Build and Publish
 
 on:
   push:
+    branches: [ 'main' ]  # Trigger on push to 'main'
```

Either this needs to be changed, or @silinternational/admins needs to change `master` branch to `main`.

```suggestion
    branches: [ 'master' ]  # Trigger on push to 'master'
```

---

Good catch, @jason-jackson. I am not sure of the ramifications either, but my gut tells me it would be ok in this case because the general public would most likely be using the Docker image and not systematically accessing files from the master branch. But it's open for discussion. In the mean time, it would be good to include both names, just so it's easier to change the branch if/when we decide to do so.

---

+1 for including both, so it will handle the rename in the future. Long term goal: everything 'main', not 'master'. When the branch is renamed, the extra parts can be removed.

---

Yeah, I'd prefer to be on `main` and not `master` here too, but it doesn't need to be right away. Good call on using both for now.